pykob/kob.py

```diff
@@ -158,7 +158,7 @@ def sounder(self, code, code_source=CodeSource.local):
             if c < -3000:  # long pause, change of senders, or missing packet
                 c = -1
 ##                self.tLastSdr = t + 1.0
-            if c > 0:  # start of mark
+            if c == 1 or c > 2:  # start of mark
```

Comment:
No, this is not because of fractions. The code element 'c' is always an integer. This case has to exclude negative numbers (which represent 'spaces', not 'marks'), zero (which should never occur), and +2 (which is the 'unlatching' code to terminate a previously latched mark).

Reply:
Okay. It's just that the 'case' statement is odd to me.